cmd/daemon/main.go

```diff
@@ -4,14 +4,21 @@ import (
 	"context"
 	"log"
 	"net/http"
+	"os"
 
 	"github.com/julz/freeze-proxy/pkg/daemon"
-	"github.com/julz/freeze-proxy/pkg/freezer"
 	authv1 "k8s.io/api/authentication/v1"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
+
+	"github.com/julz/freeze-proxy/pkg/freezer"
 )
 
+var runtimeType string
+
+func init() {
+	runtimeType = os.Getenv("RUNTIME_TYPE")
```

Comment:
any reason to do this in an init func rather than just at the top of main.go? If not I vote we drop the init func and just use a local variable

Reply:
yeah, just put it  in the main.go